src/server/CMakeLists.txt

```
@@ -170,3 +179,12 @@ set_target_properties(
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/mirserver.pc
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
 )
+
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/mirserver-internal.pc
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
+)
+
+add_custom_target(regenerate-mirserver-debian-symbols
+    COMMAND python3 ${PROJECT_SOURCE_DIR}/tools/symbols_map_generator.py --library mir_server --version 2.18 --output_symbols
```

We shouldn't be hard coding the version (especially to the wrong value)
```suggestion
    COMMAND python3 ${PROJECT_SOURCE_DIR}/tools/symbols_map_generator.py --library mir_server --version ${MIR_VERSION_MAJOR}.${MIR_VERSION_MINOR} --output_symbols
```

---

This will be `2.16`. Is that expected?

---

Hmm, that's true. I was expecting 2.17